systemd/stratis-fstab-setup-v2

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+	echo Stratis symlink with pool name and filesystem name in systemd format \
+	\(dev-stratis-\$POOL_NAME-\$FILESYSTEM_NAME\) required as an argument. >&2
+	exit 1
+fi
+
+
+if [ "$1" != dev-stratis* ]; then
+	echo Argument provided was in incorrect format. Stratis symlink with pool name \
+	and filesystem name in systemd format \(dev-stratis-\$POOL_NAME-\$FILESYSTEM_NAME\) \
+	required as an argument. >&2
+	exit 1
+fi
+
+PATH="$1"
```

Comment:
Needs to be something like `DEVPATH` to avoid overwriting `PATH` - otherwise we'll get command not found for the binaries the script runs (stratis-min and sleep).